tests/test_algorithm.py

```diff
@@ -1819,6 +1820,51 @@ def test_empty_positions(self):
         self.assertTrue(all(num_positions == 0))
         self.assertTrue(all(amounts == 0))
 
+    def test_zero_trading_days(self):
+        """
+        Test that when we run a simulation with no trading days (e.g. beginning
+        and ending the same weekend), we don't crash on calculating the
+        benchmark
+        """
+        sim_params = copy(self.sim_params)
+        sim_params.period_start = pd.Timestamp('2006-01-14', tz='UTC')
+        sim_params.period_end = pd.Timestamp('2006-01-15', tz='UTC')
+        sim_params.first_open = pd.Timestamp('2006-01-14 14:31:00', tz='UTC')
+        sim_params.last_close = pd.Timestamp('2006-01-15 21:00:00', tz='UTC')
+
+        script = """
+from zipline.api import symbol, set_benchmark
+
+def initialize(context):
+    set_benchmark(symbol('TEST'))
+    context.sid = symbol('TEST')
+
+def handle_data(context, data):
+    pass
+        """
+        algo = TradingAlgorithm(
+            script=script,
+            sim_params=sim_params,
+            env=self.env
+        )
+        algo.run(self.data_portal)
```

Comment:
I think the empty Series makes sense to me. In any case, the output of `run` is an empty DataFrame, which is understandable, since a trivial simulation is run